package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/riyacchi/eris-pagination#readme",
   "dependencies": {
+    "eris": "^0.11.2",
```

Comment:
adding eris as a dep just to use the type seems really overkill... this package doesn't actually rely on eris for functionality right?

Reply:
Yeah well.. it's going to be needed if we're going to use the paginator anyways so it wouldn't really make any difference imo

Reply:
to me it's just an inaccurate depiction of the library. the code does not depend on or use anything from eris. do the typings not work if we remove this? what if we use a peer dependency instead?

Reply:
Coming in next push